src/reducers/versions.tsx

```diff
@@ -735,6 +764,12 @@ const reducer: Reducer<VersionsState, ActionType<typeof actions>> = (
       const { selectedPath, versionId } = action.payload;
 
       const version = state.versionInfo[versionId];
+
+      if (!version) {
+        _log.error(`Version missing for versionId: ${versionId}`);
```

Comment:
I wonder if this should throw an actual error, rather than just logging. It shouldn't really be possible to have called `updateSelectedPath` without having a corresponding version loaded.